test/unit/asset-fs-inline-path-enc-es/input.js

```diff
@@ -1,4 +1,4 @@
-import fs from 'fs';
-import * as path from 'path';
+import fs from 'node:fs';
+import * as path from 'node:path';
```

Comment:
Should this test be reverted so we cover both cases?

Reply:
Yeah we can revert this one since we have the separate fixtures now, updated in https://github.com/vercel/nft/pull/287